app/graphql/types/plans/object.rb

```diff
@@ -49,6 +49,21 @@ class Object < Types::BaseObject
       field :is_overridden, Boolean, null: false
       field :subscriptions_count, Integer, null: false
 
+      def entitlements
+        # NOTE: Entitlements don't rely on the overridden plan. Unfortunately, the frontend reuse the same
+        #       component for both plan edit and subscription plan override edit.
+        #       In order to keep work minimal in the frontend for now, return the subscription entitlements instead of
+        #       in the plan entitlements when resolving a plan override. A plan override is a plan with a parent_id.
+        if object.parent_id.present?
+          subscription = object.organization.subscriptions.find_by(plan: object)
+          pps subscription
+          entitlements = ::Entitlement::SubscriptionEntitlement.for_subscription(subscription)
+          ::V1::Entitlement::SubscriptionEntitlementsCollectionSerializer.new(entitlements).serialize_models
```

Comment:
I think this is the first time we use a serializer on GraphQL, can I ask why?

Reply:
The data returned is "flat" (one feature with 2 privileges returns to rows) and the serializer is responsible for reshaping the data (grouping per feature).

I understand the surprise of seeing a serializer here, I'm happy to move this somewhere else (model or service).